tests/genomics_tests/test_genomic_job_controller.py

```diff
@@ -6,7 +6,7 @@
 from rdr_service.clock import FakeClock
 from rdr_service.dao.genomics_dao import GenomicGcDataFileDao, GenomicGCValidationMetricsDao, GenomicIncidentDao, \
     GenomicInformingLoopDao, GenomicResultViewedDao, GenomicSetMemberDao, UserEventMetricsDao, GenomicJobRunDao, \
-    GenomicMemberReportStateDao
+    GenomicMemberReportStateDao, GenomicCVLResultPastDueDao
```

Comment:
Can you update the method `test_reconcile_pdr_data` to encompass the change in the logic?

Reply:
Do you mean updating the list of `affected_tables`?  I missed that

Reply:
The tests does an insert on the records, will need to add that also, then the mock tracks the num of calls and that should be reflected in the length of tables.

Reply:
Added